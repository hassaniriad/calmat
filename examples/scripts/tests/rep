c=0
m.x=1
m.y=2
m=3
