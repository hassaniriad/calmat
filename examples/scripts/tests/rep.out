(calmat) ~> c=0
(calmat) ~> m.x=1
(calmat) ~> m.y=2
(calmat) ~> m=3

c = 0

 = 1

 = 2

m = 3
